src/common/state/redis.cc

```diff
@@ -933,6 +933,8 @@ void redis_task_table_add_task(TableCallbackData *callback_data) {
 
   TaskExecutionSpec *execution_spec = Task_task_execution_spec(task);
   TaskSpec *spec = execution_spec->Spec();
+  printf("[redis_task_table_add_task] exespec spillback_count is %d\n",
+         execution_spec->getSpillbackCount());
```

Comment:
let's remove this before merging

Reply:
yeah, all logging is temporary (for the duration of the WIP)